libpod/container_internal.go

```diff
@@ -939,7 +939,12 @@ func (c *Container) init(ctx context.Context, retainRetries bool) error {
 
 	// With the spec complete, do an OCI create
 	if err := c.ociRuntime.createContainer(c, nil); err != nil {
-		return err
+		if err := c.ociRuntime.createContainer(c, nil); err != nil {
```

Comment:
this seems like we're trying to create the container twice

Reply:
Agree - we should not need the second call

Reply:
not sure what happened here. some sort of patch issue.  correcting!